yaksh/models.py

```diff
@@ -68,21 +63,29 @@
                 ('completed', 'Completed'),
               )
 
+
 def get_assignment_dir(instance, filename):
-    return '%s/%s/%s' % (instance.user.user, instance.assignmentQuestion.id, filename)
+    return os.sep.join((
+        instance.user.user, instance.assignmentQuestion.id, filename
```

Comment:
I know it was this way before. I commented on it because I had a painful experience of finding the assignment files. The problem is these question IDs are not displayed anywhere on the interface except for the URLs. Maybe as a quick fix we can display the question IDs on the interface somewhere. Anyways this is mergable.